src/view/prsTreeDataProvider.ts

```diff
@@ -70,6 +70,8 @@ export class PullRequestsTreeDataProvider implements vscode.TreeDataProvider<Tre
 					return vscode.commands.executeCommand('workbench.action.openSettings', `@ext:${extensionId} queries`);
 				case 'Configure Remotes...':
 					return vscode.commands.executeCommand('workbench.action.openSettings', `@ext:${extensionId} remotes`);
+				case 'Sign out of GitHub...':
```

Comment:
I think it would be better to either create a new command, or conditionally include this. Otherwise, even if you're not signed in and you click the gear on the PR tree, you see an option to sign out, which seems unexpected

Reply:
Oh, good point.

Reply:
Fixed